doc/user_guide/encryption.rst

```diff
@@ -22,11 +22,16 @@ Certification verification
 --------------------------
 
 - Exasol running "on-premises" uses self-signed SSL certificate by default. You may generate a proper SSL certificate and upload it using `instruction <https://docs.exasol.com/administration/on-premise/access_management/tls_certificate.htm>`_.
-- Exasol Docker uses self-signed SSL certificate by default. You may generate a proper SSL certificate and use it via editing of EXAConf file. More details are available on the `GitHub page <https://github.com/exasol/docker-db>`_.
-- Exasol SAAS running in the cloud uses proper certificate generated by public certificate authority. It does not require any extra setup.
-
-(security risk) To disable strict certification verification, set ``websocket_sslopt`` to ``{"cert_reqs": ssl.CERT_NONE}``.
-This might be needed for specific use cases (e.g. Docker container, "on-premises" Exasol setup).
+- Exasol Docker uses self-signed SSL certificate by default. You may generate a proper SSL certificate and submit it for use via the ConfD API. More details are available on:
+   - `GitHub for Exasol Docker <https://github.com/exasol/docker-db>`_
+   - `ConfD API <https://docs.exasol.com/db/latest/confd/confd.htm>`_
+   - `confd_client cert_update <https://docs.exasol.com/db/latest/confd/jobs/cert_update.htm>`_
+- Exasol SAAS running in the cloud allows users to authenticate using:
+   1. `username & personal access token (PAT) <https://docs.exasol.com/saas/administration/access_mngt/access_token.htm>`_
+   2. `using OpenID <https://docs.exasol.com/db/latest/sql/create_user.htm#Authenti4>`_
```

Comment:
OpenID is not specific to SaaS, maybe make it a separate section

Reply:
For OpenID the SSL certificate seems to depend on the DB you connect to, I think. https://exasol.my.site.com/s/article/Changelog-content-9170?language=en_US states that you send the access token to the DB and DB compares it to the keys it got from the authentication server.

Reply:
You also could reference the Exasol SaaS Docu here:
https://docs.exasol.com/saas/home.htm

There is also a page about Access management:
https://docs.exasol.com/saas/administration/access_mngt/access_management.htm